src/Html.php

```diff
@@ -1384,21 +1527,45 @@ public static function ol($items, array $options = []): string
      * call. This method will take out these elements, if any: "prompt", "options" and "groups". See more details in
      * {@see dropDownList()} for the explanation of these elements.
      *
+     * @psalm-param iterable<array-key, string>|string|null $selection
+     * @psalm-param array<array-key, array|string> $items
+     *
      * @throws JsonException
      *
      * @return string the generated list options
      */
-    public static function renderSelectOptions($selection, array $items, array &$tagOptions = []): string
+    public static function renderSelectOptions($selection, array $items, array $tagOptions = []): string
```

Comment:
What's the reson for this change?

Reply:
Usability. Without this change 3th argument should be variable, can't use method like this: 
`Html::renderSelectOptions($selection, $items, ['encode' => false])`